.env.development

```diff
@@ -1,2 +1,2 @@
-BASE_URL="https://areweheadlessyet.staging.wagtail.org/"
-INSTANCE="staging"
\ No newline at end of file
+BASE_URL="http://0.0.0.0:8000/"
```

Comment:
Could we add a note in the README.md to say that local development for the FE requires running a BE? (+ how to set this up) For my local testing purposes, I just used the .env.production BASE_URL here instead.

Reply:
I've added that in this PR: https://github.com/wagtail/areweheadlessyet/pull/14. Fancy reviewing it too :stuck_out_tongue_winking_eye:?